legal-api/src/legal_api/services/authz.py

```diff
@@ -82,33 +116,9 @@ def authorized(  # pylint: disable=too-many-return-statements
         if any(elem in action for elem in staff_only_actions):
             return False
 
-        template_url = current_app.config.get('AUTH_SVC_URL')
-        auth_url = f'{template_url}/entities/{identifier}/authorizations'
-
-        token = jwt.get_token_auth_header()
-        headers = {'Authorization': 'Bearer ' + token}
-        try:
```

Comment:
moved this into `_call_auth_api` so that it could be reused in `get_account_products`